src/JabbR-Core.Data/Models/UserRoomAllowed.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace JabbR_Core.Data.Models
 {
-    public partial class ChatRoomChatUserOwner
+    public partial class UserRoomAllowed
```

Comment:
Looking at PR, weren’t we going to use the names ChatRoomChatUserAllowed and ChatRoomChatUserOwner (so its more in line with what EF shows)? If we’re going to rename away from the EF many-many style (no prob with that), may I suggest we simplify but with intent (does this represent those tables appropriately?)
ChatRoomUsers
ChatRoomOwners